meta-mender-demo/conf/layer.conf

```diff
@@ -17,4 +17,4 @@ IMAGE_OVERHEAD_FACTOR = "1.0"
 IMAGE_FEATURES += "splash"
 
 LAYERSERIES_COMPAT_mender-demo = "warrior"
-LAYERDEPENDS_mender-demo = "mender"
+LAYERDEPENDS_mender-demo = "mender-client"
```

Comment:
This "mender" reference is for the layer as a whole, including mender-artifact, so this should definitely be just "mender".

Same goes for the other `LAYERDEPENDS` changes.

Reply:
I guess that answers the above. Changing.